### PR TITLE
update ui

### PR DESCRIPTION
**Pull request by MeisterJustice**
This PR does some ui improvements. Please check https://www.notion.so/metastablelabs/Mobile-Cleanup-Integration-120716767cb480b88f48e1263cbd17b1

The file changes are getting large so I'll cut this PR here. Changes to all font sizes were made. We're currently adding `font-size stylings` each time we need to(which caused the large file size). This will change. To move faster, we didn't implement and setup typography that can be reused from the get go. A PR will be raised soon to fix that.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "slug": "liquid",
     "version": "1.0.0",
     "orientation": "portrait",
-    "icon": "./assets/images/icon.png",
+    "icon": "./assets/images/liquid-icon.png",
```
Comment (renanmav) [inline app.json:7]: @MeisterJustice you need to run the prebuild command to update ios and android folders, no?

Comment (MeisterJustice) [reply app.json:7]: what's the command please? i believe i did @renanmav

Comment (renanmav) [reply app.json:7]: @MeisterJustice I don't see changes in folders `ios` and `android` in the diff.

Commands are:

```bash
npx expo prebuild -p ios
npx expo prebuild -p android
```

Comment (MeisterJustice) [reply app.json:7]: thank you

Comment (MeisterJustice) [reply app.json:7]: done @renanmav

**Review by njokuScript — APPROVED** [review]
LGTM

**Review by renanmav — APPROVED** [review]

**Pull request merged**